### PR TITLE
Improve error messages in space-age exercise

### LINKED ISSUE #67: Improve error messages in space_age exercise
Related to [this comment](https://github.com/exercism/gdscript/pull/66#discussion_r1605451162).

The `space-age` exercise uses a method called `roughly_equal` to compare float values. If the values are not "roughly equal", the resulting error message will say "False is not True". It would be more intuitive to show actual values in the error message.

### PR DESCRIPTION
**Pull request by pfertyk**
Fixes #67 .

### CODE REVIEW

**Comment by pfertyk** [comment]
@BNAndras I think this will help with obscure error messages in space-age exercise ;) Could you please take a look?

File: exercises/practice/space-age/space_age_test.gd
Comment (BNAndras) [inline exercises/practice/space-age/space_age_test.gd:1]: Looks good. Do we need to round the expected values though? I think we can just round the calculated age. That should make it a bit clearer what's going on hopefully.

Comment (pfertyk) [reply exercises/practice/space-age/space_age_test.gd:1]: The last example was raising an error, saying that the expected value was 0.35, and the actual one 0.35  ... :thinking: So, just to be sure, I decided to round all the values :stuck_out_tongue: That took care of the error :wink:

Comment (BNAndras) [reply exercises/practice/space-age/space_age_test.gd:1]: Fair enough

**Review by BNAndras — APPROVED** [review]

**Pull request merged**